### PR TITLE
Added notes router, mode, table and updated Readme Installation notes

### PR DESCRIPTION
**Pull request by charisearter**
## Description ##

Clearly documents the reason for the pull request and what has changed. Delete this test and add your own.
- [ ] This is a bugfix.
- [X] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?

## Review ##

- [X] Have you added two reviewers to this pull request?
- [X] Is this a work in process?
- [x] Is this ready to be reviewed?

### CODE REVIEW

File: api/notes/notesRouter.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const Notes = require('./notesModel.js');
+
+const router = express.Router();
+
+router.get('/notes', (req, res) => {
+  Notes.getNotes()
+    .then((notes) => res.status(200).json(notes))
+    .catch((error) => {
+      console.log('error in Note Router GET', error);
```
Comment (Joe-Bay) [inline api/notes/notesRouter.js]: Overall it looks really clean, I would just clean up some of the console logs that may be unnecessary on the main branch.

Comment (charisearter) [reply api/notes/notesRouter.js]: Doing it now. Thanks for reminding me.

**Review by hnetty — APPROVED** [review]

File: data/migrations/20201109081432_create-watchlist.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
+  return knex.schema
+    .createTable('watchlist', (list) => {
+      list.increments();
+      list.string('list_title', 50).unique().notNullable();
+      list.string('profile_id').unsigned().references('id').inTable('profiles');
+      list
```
Comment (VinceWilliamsDev) [inline data/migrations/20201109081432_create-watchlist.js]: It looks like this is creating a 1-1 relationship between lists and projects. So a list will only be able to contain a single bridge. We need a 1-many relationship between the list and bridges.

**Comment by hnetty** [comment]
Additional comments and such

**Review by VinceWilliamsDev — CHANGES_REQUESTED** [review]
Overall looks really good! Need to think about the relationship between lists and bridges though.

File: data/migrations/20201109081432_create-watchlist.js (hunk 2 above)
Comment (charisearter) [reply data/migrations/20201109081432_create-watchlist.js]: Thank you! I wanted to talk with you about it later.

**Review by VinceWilliamsDev — APPROVED** [review]
Nice work on all the changes, pair programming really helped

**Comment by VinceWilliamsDev** [comment]
Good work, I think this should work well.

**Pull request merged**